packages/affiliates/gas-rebate/rebates/Rebate_25.json

```diff
@@ -0,0 +1,380 @@
+    "0xd7925984736824b4aecf8301c9aee211cd976494": 5.06405,
+    "0xfec2ade141272f5271febadfca5d21ca30fc04a5": 10.8288625,
+    "0xb3cbbc13106d3e1f1f68a7b7a19e8c69e3c3aca4": 10.735325,
+    "0xf445567e961c533e26430005c142dee831a6376e": 13.0030125,
+    "0x7f1c6e55c5bec9a47a3a7f4c17d2d31d38ddc2aa": 4.252175,
+    "0x239b74c2e5505634b7c38c8eeb1de385a16e24e2": 11.0511,
+    "0xea7c2df8ab95bf0af91f8e4034b493733aca21df": 15.652662499999998,
+    "0x7e367235ef80109531113d8284a5593ba6a40c22": 9.7772,
+    "0x384d91e6b8d032ab9d844eb58b85df6543e616f0": 15.5300875,
+    "0x8348c5ec31d486e6e4207fc0b17a906a0806550d": 15.0265875,
+    "0x0bfd7ae78ed2b89f7e7d85938e9317b5d494047b": 11.05125,
+    "0x34d35d8846a071bcbd011f728b0d3f52fa375b07": 3.2823875,
+    "0x30daba789bf9390a75d6b4f0efd21eff03f16c19": 8.8145875,
+    "0x2e47e3f888d561c28ffa2a7fe3a2b2794778de0a": 7.4459625,
+    "0x9a97c80956bce4dd87c2df6a93900dbfcbf0d60a": 10.2177375,
+    "0x30ba060f0b4ead43e24593bf71e907f397baf532": 12.33465,
+    "0xe6fcb51c7ee5d15a5c8fc2b23d167db6a963c799": 5.383775,
+    "0x01371b3ff34e1bbcf599aede17e3f211f6974953": 5.525475,
+    "0x9b6f4d48fd61313204f7c8fbe9dfec6384aa4ced": 2.1262375,
+    "0x4cbacd6821b53ea9a41e8383303b8a78e3a9edfa": 16.5494375,
+    "0x336b9de28d67d692c31df93a9b8630b2fa50d88f": 15.335887499999998,
+    "0xb0903a62f5ca7a215aa67232ba691cb801decd5b": 15.478337499999999,
+    "0x4b070c428ddbb49524abf2d4be4998c1e3b6f3a5": 15.4786375,
+    "0xabbcf7b0735e4eb152fd13fa02691cc046fac0bd": 9.026412500000001,
+    "0x66b5409f52d23ef87bdbfaa7312f8d790755fe13": 10.353737500000001,
+    "0x49e0cdd718d330e5a27856256236b4e56a76d3fb": 15.6523625,
+    "0xc3d26aa4954816330c5ffc9bb2d46acbceaff130": 15.6526625,
+    "0x2cf6988d57d0df95d3ff37cdfbfe0520f15c860f": 15.303112500000001,
+    "0x8a6800c753ed8ed19e0df473a97a7a94b2cecf54": 6.38025,
+    "0x96c00fdda3f3129a55c0bc9b48ef504c3d5594e4": 14.987237499999999,
+    "0x3d88456277e32d62575267c3564b72387c3f21f7": 15.5100625,
+    "0x1be514893bfd6f13f508a71a37b0246ed7b4bb8a": 15.304162499999999,
+    "0x9e3a4ed4eea5c27a5e82a32e90482a4827a4ae80": 15.477887500000001,
+    "0x3d65ed138686e2ad54006d4db83ad4597d645370": 12.334800000000001,
+    "0x87d1465044ddb9bf2e5fb189067cb340a7b0074a": 8.8914375,
+    "0x969397c71eb51f7fdcf6527db4d97871b05d6f63": 9.0261125,
+    "0x0a01f4a6711012157c1b33ad9c01af7f526962f5": 11.195675,
+    "0x1fdef3d3c7984a945008c253caa6cf380fddb16f": 14.9553625,
+    "0xdcf99ba641c98e37df8ea31188ad7e7fa5501b09": 15.6526625,
+    "0x3b271e8f1409e8e90c484d3591143bd7e1d2a177": 15.6523625,
+    "0x11efc2638184a0382bf1bb29c4047c6111f8f3c7": 15.6525125,
+    "0x7cac2ab1cf9dc459ca88a85d72ac94de2691f1e5": 15.6517625,
+    "0x82b6a4ebc3904b7aabd4e85510045eff0d43a6bd": 9.026562499999999,
+    "0x469f91dc3253165f52bb56eaa4aee838c1c7822d": 12.547575,
+    "0xacd3dc5c6f9cb967b5e1f719001e85039b0c976f": 12.903400000000001,
+    "0xbca0fd352dce62fa1b7c7f9fa8f0c1de7fef09d9": 13.613199999999999
+  },
+  "priceData": {
```

Comment:
Gas prices and UMA prices all look constant? Are we not doing lookups for these anymore?

Reply:
We don't use premium etherscan for UMA/ETH prices, though, do we? https://github.com/UMAprotocol/protocol/blob/b80ce3919116cf7cf0eebd24c6c057f32f2f4bed/packages/affiliates/gas-rebate/VoterGasRebate.js#L528-L544

Reply:
it's meant to come from coingecko.  I'm not totally sure why the price of this is wrong. the hardcoded value is pretty close to what it's been for most of this month. I can spend time looking into this but not sure how useful it is given it's pretty close to the "true" price.

Reply:
I think we should revamp this for next time, especially with DVM 2.0 coming up!